### DISCUSSION TITLE
Can I put shortcuts to non-scoop apps in the Scoop Apps folder?

### DISCUSSION THREAD

**Discussion by jasonistre** [ScoopInstaller/Scoop#5779] [Help and Questions] [answered]
Can I place custom shortcuts to applications I didn't install with scoop in the "Scoops Apps" start menu folder just so I can have some tools I use located in the same place, or is this folder and its contents managed by scoop making additional files such as .lnk, .ps1, .bat, ,txt I place there problematic?

**Comment by HUMORCE** [comment, marked as answer]
Scoop only adds/removes the corresponding Shortcut during (un)installation of app. But I'm not sure if the whole directory will be deleted when uninstalling Scoop.

This depends on how you use startmenu. For me: I just searching shortcut in startmenu, so I don't care which direcotry that the shortcut stored.

You can also create shim via `scoop shim` to access your scripts, but I recommend that just add a specific dir to the PATH directly.
